courses/pyladies/plan.yml

```diff
@@ -1,67 +1,99 @@
 - name: Instalace
   materials:
-  - link: lessons/beginners/cmdline
-  - link: lessons/beginners/install
-  - link: lessons/beginners/install_editor
-  - link: lessons/beginners/install_git
+  - name: Úvod do příkazové řádky
```

Comment:
Why repeat the name here? If this links to a lesson, and we just want to use the title of the lesson, it should be possible to omit `name`.